### PR TITLE
Update 1

### PR DESCRIPTION
**Pull request by makeclean**
This PR should address #68 #64 #63 #59 #58 #57 #56 #27

### CODE REVIEW

**Comment by makeclean** [comment]
Should now also address #9

File: src/report.tex [hunk 1]
```diff
@@ -1160,7 +1168,7 @@ \subsection{Neutron Activation}
 using \gls{alara_c} 2.9.1RC. There was an activation calculation for each 
 neutron mesh and case, therefore
 there were a total of fourteen activation calculations performed. The activation
-calculations used a uniform `truncation tolerance', a parameter in \gls{alara_c}
+calculations used a uniform `truncation tolerance' of $1.0\times10^8$, a parameter in \gls{alara_c}
```
Comment (gonuke) [inline src/report.tex]: isn't this 1e-8?

File: src/report.tex [hunk 2]
```diff
@@ -2100,6 +2114,9 @@ \subsubsection{Breakdown of Cross Talk}
 
 \newpage
 \clearpage
+
+\newpage
+\clearpage
```
Comment (gonuke) [inline src/report.tex]: two newpages in a row?

File: src/report.tex [hunk 3]
```diff
@@ -2143,10 +2160,30 @@ \subsection{Conclusion}
 activation in the region. It is suggested that the front 50 cm of the B$_4$C
 drawers could be removed, reducing the weight of the port plug without
 significantly impacting the shielding capability.
-
-%\newpage
-%\clearpage
-%\section{Acceptance Criteria}
+\subsection{Estimation of Uncertainties}
```
Comment (gonuke) [inline src/report.tex:2164]: none of these mention the statistical uncertainty of the neutron transport which we also can't propagate

File: src/report.tex [hunk 4]
```diff
@@ -2176,6 +2213,23 @@ \subsection*{High Throughput Methodology}
 produced, but increasing the desired number of particle histories could see this 
 grow to 20 Tb of space.
 
+\subsection*{ Transport Improvments}
```
Comment (gonuke) [inline src/report.tex]: Improvements

File: src/report.tex [hunk 5]
```diff
@@ -2176,6 +2213,23 @@ \subsection*{High Throughput Methodology}
...
+\subsubsection*{Further Neutron Histories}
+This calculation ran a specific number of neutron histories which was limited by
+time contraints. Future calculations should run a greater number of neutron histories
+which would reduce the statistical noise in the \gls{sdr} photon sources and allow
+smaller uncertainty bars to be placed on the neutron sources
```
Comment (gonuke) [inline src/report.tex]: uncertainty bars on photon sources??? not neutron sources?

**Comment by makeclean** [comment]
And #62

File: src/report.tex (hunk 1 above)
Comment (makeclean) [inline src/report.tex]: Quite right

**Comment by gonuke** [comment]
This looks good now... let me know if you are doing more or are ready to merge

**Comment by gonuke** [comment]
Thanks - nearly there.

**Comment by gonuke** [comment]
Thanks - nearly there

**Pull request merged**